src/project.ts

```diff
@@ -262,7 +273,7 @@ class Project extends ServiceObject {
        */
       setMetadata: {
         reqOpts: {
-          method: 'PUT',
+          method: 'PUT' as METHOD_TYPE,
```

Comment:
Why do we need to cast this? Did this change in the common? I feel like if thats the case they should export the type.